KERNEL_BUILD_DOCUMENTATION.md

```
@@ -11,7 +11,7 @@ This comprehensive documentation details every function and process in the `buil
 3. [Dependency Management](#dependency-management)
 4. [Toolchain Setup](#toolchain-setup)
 5. [Repository Management](#repository-management)
-6. [Sukisu Ultra Integration](#sukisu-ultra-integration)
+6. [KernelSU-Next Integration](#ksun-integration)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)